### PR TITLE
Webin_cli API Docs

### PR DESCRIPTION
**Pull request by ahmadazd**
This is the documentation for the Webin-cli SARS-COV-2 genome submission API to be included within the SARS-COV-2 readthedoc.

### CODE REVIEW

File: help_and_guides/Webin-Cli_SARS-CoV-2_Genome_Submission_REST_API.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+]
+
+## Here we used the validation test(dev) URL
+server = "https://wwwdev.ebi.ac.uk/ena/submit/webin-cli/api/v1/genome/covid-19/validate"
+
+for sample in data:
+    sample_json = json.dumps(sample)
+    response = requests.post(
+        server, headers={"accept":"application/json", "Content-Type":"application/json"}, 
+        data=sample_json, auth=('Webin-56388', 'IlR07wf18uaE')
```
Comment (carlacummins) [inline help_and_guides/Webin-Cli_SARS-CoV-2_Genome_Submission_REST_API.md]:
```suggestion
        data=sample_json, auth=('Webin-XXXXXX', 'password')
```

**Review by raskoleinonen — CHANGES_REQUESTED** [review]
- Suggested several changes. Please see branch raskoleinonen-patch-1.

**Review by raskoleinonen — APPROVED** [review]

File: help_and_guides/Webin-Cli_SARS-CoV-2_Genome_Submission_REST_API.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
+# Webin SARS-CoV-2 Genome Submission Web API
+
+## Introduction
+
+Webin SARS-CoV-2 Genome Submission Web API is a JSON based service used to submit SARS-CoV-2 genome assemblies to the European Nucleotide Archive (ENA).
```
Comment (nadimm-rahman) [inline help_and_guides/Webin-Cli_SARS-CoV-2_Genome_Submission_REST_API.md]:
```suggestion
Webin SARS-CoV-2 Genome Submission Web API is a JSON based service used to submit SARS-CoV-2 genome assemblies to the European Nucleotide Archive (ENA). For further information on submitting SARS-CoV-2 genomes, see our [SARS-CoV-2 Submission Instructions](https://ena-browser-docs.readthedocs.io/en/latest/help_and_guides/sars-cov-2-submissions.html#submitting-assemblies).
```

**Review by nadimm-rahman — CHANGES_REQUESTED** [review]
Recommending addition of a link to the SARS-CoV-2 submission instructions. We should also add a link to this page from the SARS-CoV-2 submission instructions page.

**Pull request merged**